udata/core/dataset/events.py

```diff
@@ -1,12 +1,15 @@
 import datetime
 from flask import current_app
+from udata_event_service.producer import produce
```

Comment:
I think it's not needed, except if we want to encapsulate udata_event_service everywhere, in order to do:
```
from udata.event import produce
```

Reply:
Ah sorry, I meant in [udata/event/__init__.py](https://github.com/opendatateam/udata/pull/2743/files#diff-f84ff018df265711184d54da187835a924f3f64d7a84352f84cbc09586a8f584)

Reply:
I was just replacing the previous produce but unsure whether any of the two imports of that file is actually useful.